javascript-modal/style.css

```diff
@@ -0,0 +1,75 @@
+body {
+  width: 1200px;
+  height: auto;
+  position: relative;
+  margin: 0;
+}
+
+.open-button {
+  margin-top: 0.5rem;
+  margin-left: 0.5rem;
+  width: 150px;
+  font-size: 20px;
+  color: white;
+  background-color: royalblue;
+  box-shadow: 2px 2px 2px 2px rgb(182, 179, 179);
+  border-radius: 5px;
+}
+
+.open-button:hover {
+  cursor: pointer;
+}
+
+.hidden {
+  height: 100vh;
+  width: 1200px;
+  text-align: center;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  transform: scale(0);
```

Comment:
This is not the preferred way to make an element not visible. You can get rid of all of the styling for this rule, and use `display: none` to get the hidden effect 👍